superbench/benchmarks/micro_benchmarks/cublaslt_function.py

```diff
@@ -75,12 +89,12 @@ def _preprocess(self):
                 f' -a -W {self._args.num_warmup_autotune}'
                 f' -I {self._args.num_steps_autotune}'
             ) if self._args.enable_autotune else ''
-
-            self._commands.append(
-                f'{self.__bin_path} -m {_m} -n {_n} -k {_k} -b {_b} '
-                f'-w {self._args.num_warmup} -i {self._args.num_steps} -t {_in_type}'
+            command = f'{self.__bin_path} -m {_m} -n {_n} -k {_k} -b {_b} ' + \
+                f'-w {self._args.num_warmup} -i {self._args.num_steps} -t {_in_type}' + \
                 f'{(" " + autotune_args) if autotune_args else ""}'
-            )
+            if self._args.enable_ncu_profiling:
+                command = f'ncu --set full --launch-skip {self._args.num_warmup - 1} --launch-count 1 --csv ' + command
```

Comment:
For these kind of flags which may be used by other benchmarks, should we consider put it into base class or into sb runner? Hongtao and Shenghsun are adding `nsys profile` for benchmarks, they get the enable flag from environment variables and append `nsys profile` in runner.py file

Reply:
Hi guoshuai, I think ncu is not that common use as nsys, which only target specific kernel, so I'm not sure which other benchmark would use it and how the command would change to adapt, as you see, I need to leverage benchmark argument in my current usage, so I can not put it into runner